services/QuillLMS/client/app/bundles/Evidence/components/studentView/promptStep.tsx

```diff
@@ -291,7 +298,9 @@ export class PromptStep extends React.Component<PromptStepProps, PromptStepState
     } else if (this.unsubmittableResponses().includes(entry) || awaitingFeedback) {
       className += ' disabled'
       onClick = () => {}
-    } 
+    } else if(responseOverCharacterLimit) {
+      className += ' disabled'
+    }
```

Comment:
I think you probably also want an `onClick` here or else it will still submit - like the one we have in the clause above. At which point this statement is identical to the clause above, so I'd just add `|| responseOverCharacterLimit` to the conditional.

Reply:
good catch!